src/simulation/m_mpi_proxy.fpp

```diff
@@ -552,6 +552,16 @@ contains
             end if
         end do
 
+        ! Update the min and max of the cells in each direction
+        mn_max = max(m, n)
+        np_max = max(n, p)
+        mp_max = max(m, p)
+        mnp_max = max(m, n, p)
+        mn_min = min(m, n)
+        np_min = min(n, p)
+        mp_min = min(m, p)
+        mnp_min = min(m, n, p)
```

Comment:
it looks like you're putting this in every file, but if you put it in one file the others will see it...